requirements/dev.txt

```diff
@@ -15,6 +15,8 @@ parameterized~=0.7.4
 pathlib2>=2.3.5; python_version < '3'
 click~=7.1
 dateparser~=0.7
+pillow~=8.1
```

Comment:
Not compatible with python 2, 6.2.2 was the last one

Reply:
will update it with pillow ~=6.2